src/PwrDrvr.LambdaDispatch.Router/LambdaInstanceManager.cs

```diff
@@ -87,7 +87,7 @@ public class LambdaInstanceManager : ILambdaInstanceManager
   private readonly IMetricsLogger _metricsLogger;
 
   private readonly Channel<LambdaInstanceCapacityMessage> _capacityChannel
-    = Channel.CreateBounded<LambdaInstanceCapacityMessage>(new BoundedChannelOptions(100)
+    = Channel.CreateBounded<LambdaInstanceCapacityMessage>(new BoundedChannelOptions(1)
```

Comment:
This 1 might not be the best... so far it's not been an issue, but this may need review